package.json

```diff
@@ -27,6 +27,7 @@
     "precommit": "lint-staged"
   },
   "dependencies": {
+    "@reach/listbox": "^0.15.2",
```

Comment:
I checked the accessbility and I was surprised how perfect the implementation worked. Then I saw that you have pulled in reach ui. Superb library and I have seen that they indeed put a focus on accessibility.

I checked how the dropdown opens/closes, how you can navigate and how the labels are rendered in the accessibility object model👌